controllers/grid/files/copyedit/CopyeditedFilesGridDataProvider.inc.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file controllers/grid/files/copyedit/CopyeditedFilesGridDataProvider.inc.php
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class CopyeditedFilesGridDataProvider
+ * @ingroup controllers_grid_files_copyedit
+ *
+ * @brief Provide access to copyedited files management.
+ */
+
+
+import('lib.pkp.controllers.grid.files.SubmissionFilesGridDataProvider');
+
+class CopyeditedFilesGridDataProvider extends SubmissionFilesGridDataProvider {
```

Comment:
Copyedited or copyedit files? Copyedited feels like they are already copyedited, which I think it's not the case right?